### DISCUSSION TITLE
Speedtest-Tracker

### DISCUSSION THREAD

**Discussion by Vakrehus** [tteck/Proxmox#823] [Ideas]
# Speedtest Tracker
https://github.com/henrywhitaker3/Speedtest-Tracker

This program runs a speedtest check every hour and graphs the results.

![Screenshot of Speedtest-Tracker](https://user-images.githubusercontent.com/36062479/78822484-a82b8300-79ca-11ea-8525-fdeae496a0bd.gif)

I have had the Speedtest tracker running very reliably on Docker on a small x86 server now for over a year.
The script has helped me a lot, to find problems with my ISP and report them to them.
Now I have to move to Proxmox and I don't want to take Docker with me again, but rather have the script running natively in an LXC.

I can very well imagine that the community has a keen interest in this.

**Comment by Vakrehus** [comment]
That tracker above is no longer in active development, so I switched to this one, which runs very well in an Docker LXC.

https://github.com/alexjustesen/speedtest-tracker

![Screenshot Speedtracker](https://raw.githubusercontent.com/alexjustesen/speedtest-tracker/main/.github/screenshots/dashboard_screenshot.png)

**Reply by Mrdindon** [reply]
would you have any tutorial for beginners on how to install it via lxc ?

**Reply by Vakrehus** [reply]
Sorry, my fault. Speedtest-Tracker is running as Dockercontainer in a LXC.

**Reply by BeerBelly** [reply]
Having issues getting this to start in a privileged Docker LXC. If I leave the user as root the logs say to change the user and if I switch to a different one the init script fails....anyone able to help please?
